plugins/unicodelib/unicodelib.cpp

```diff
@@ -718,6 +719,43 @@ unsigned unicodeEditDistanceV4(UnicodeString & left, UnicodeString & right, unsi
     return da[mask(leftLen-1)][rightLen-1];
 }
 
+UnicodeString excludeNthWord(RuleBasedBreakIterator& bi, unsigned textLen, UChar const * text, unsigned n)
+{
+    UnicodeString source(text, textLen);
+    bi.setText(source);
+    int32_t idx = bi.first();
+    int32_t wordidx = 0;
+    unsigned wordBeginning = 0;
+    while (idx != BreakIterator::DONE) {
```

Comment:
style:  The general style is for curly brackets to be on the following line.  The function you started from doesn't follow that style.   So while, if, do in this function should be on a new line.